README.md

```diff
@@ -19,6 +19,7 @@
 ### 2023
 | Author(s) | Title | Venue |
 |:--------- | ----- | ----- |
+| Floridi | [Machine Unlearning: its nature, scope, and importance for a “delete culture”](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4455976) | SSRN |
```

Comment:
Can you please change "SSRN" to the actual journal name, I think in this case it is "Philosophy & Technology".

Reply:
Done! Thanks!